.vscode/launch.json

```diff
@@ -28,6 +28,46 @@
             "cwd": "${workspaceFolder}/server",
             "console": "externalTerminal"
         },
+        {
+            "name": "Launch Server (gdb)",
+            "type": "cppdbg",
+            "request": "launch",
+            "preLaunchTask": "cargo build",
+            "program": "${workspaceFolder}/server/target/debug/odoo_ls_server",
+            "args": ["--use-tcp"],
+            "cwd": "${workspaceFolder}/server",
+            "externalConsole": false,
+            "MIMode": "gdb",
+            "miDebuggerPath": "/usr/bin/rust-gdb",
```

Comment:
For me rust-gdb was not installd under `/usr/bin/rust-gdb` but rather under `~/.cargo/bin/rust-gdb`.  
Regardless setting it to `"miDebuggerPath": "rust-gdb",` worked for me as rust-gdb is in `PATH`